packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -785,7 +676,7 @@ describe.skip( 'Navigation', () => {
 		expect( isScriptLoaded ).toBe( true );
 	} );
 
-	describe( 'Creating and restarting', () => {
+	describe.skip( 'Creating and restarting', () => {
```

Comment:
did this break?

Reply:
I don't think so, but they also weren't enabled in trunk.

This branch predated these tests and I'd already removed most of the mocking utilities, so when I rebased I realised I had a lot more work to do 😄 

I'll take a look at switching them away from mocks and over to using the REST API in a fresh PR.